src/init/starship.nu

```diff
@@ -6,13 +6,16 @@ let-env PROMPT_MULTILINE_INDICATOR = (^::STARSHIP:: prompt --continuation)
 # TODO: Also Use starship vi mode indicators?
 let-env PROMPT_INDICATOR = ""
 
+# Render the right prompt on the last line of the prompt.
+let-env config = ($env.config | upsert render_right_prompt_on_last_line true)
```

Comment:
This will fail if the user hasn't set `$config` before sourcing the file.

Reply:
Sorry for the late response. I've checked the version and `$env.config`.